runconfig/hostconfig.go

```diff
@@ -249,6 +249,7 @@ type HostConfig struct {
 	RestartPolicy     RestartPolicy         // Restart policy to be used for the container
 	SecurityOpt       []string              // List of string values to customize labels for MLS systems, such as SELinux.
 	ReadonlyRootfs    bool                  // Is the container root filesystem in read-only
+	Sysctls           []string              // List of system controls to be set in the container
```

Comment:
This I would still prefer to see at least as a map[string]string rather than splitting on `=` to hand off to the exec driver.
Thoughts from anyone else?